src/prpy/perception/kinbody_helper.py

```diff
@@ -0,0 +1,87 @@
+
+def transform_to_or(kinbody, detection_frame, destination_frame, 
+                    reference_link=None):
+    """
+    Transform the pose of a kinbody from a pose determined using TF to 
+    the correct relative pose in OpenRAVE. This transformation is performed
+    by providing a link in OpenRAVE that corresponds directly to a frame in TF.
+
+    @param kinbody The kinbody to transform    
+    @param detection_frame The tf frame the kinbody was originally detected in
+    @param destination_frame A tf frame that has a direct correspondence with
+      a link on an OpenRAVE Kinbody
+    @param reference_link The OpenRAVE link that corresponds to the tf frame
+      given by the destination_frame parameter (if None it is assumed
+      that the transform between the OpenRAVE world and the destination_frame 
+      tf frame is the identity)
+    """
+
+    import numpy, tf, rospy
```

Comment:
FYI guys, multiple `import`s on a single line is _not_ PEP8 :neckbeard: 
https://www.python.org/dev/peps/pep-0008/#imports